add preset support / animated gif

simonbuehler opened the pull request:
allow to use presets

simonbuehler committed:
add preset support

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/hudl/HudlFfmpeg?pullRequest=96) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/hudl/HudlFfmpeg?pullRequest=96) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/hudl/HudlFfmpeg?pullRequest=96) it.</sub>

simonbuehler committed:
Animated Gif

- add support for rgba and lavfi formats
- add Gif tyep as Video
- add Pattegen/Palleteuse filters for optimizing gif palettes
- add 420p Format

simonbuehler committed:
add Curves filter, FrameRateInput (-framerate predeclared variant), fix FormattingUtility

simonbuehler committed:
fix build errors

simonbuehler committed:
move preset to Settings dir

Casey-Bateman commented:
I will merge and cut into a new release 2/1/2019.

simonbuehler commented:
signe the cla, no idea why its still in pending state

simonbuehler committed:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	Hudl.FFmpeg/Hudl.Ffmpeg.csproj

simonbuehler committed:
Merge branch 'hudl:master' into master

simonbuehler committed:
fix Naming

simonbuehler committed:
Merge branch 'master' of https://github.com/simonbuehler/HudlFfmpeg

simonbuehler commented:
hi, is this package still maintained? pls merge

simonbuehler commented:
@Casey-Bateman ping

sasnyk-hudl commented:
![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### :no_entry: **Snyk checks have failed. 25 issues have been found so far.**

| Icon     | Severity | Issues  |
| -------- | -------- | ------- |
| ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-critical.svg) | Critical | 0 |
| ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg) | **High** | **25** |
| ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg) | Medium | 0 |
| ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-low.svg) | Low | 0 |

:no_entry: **security/snyk** check is complete. 25 issues have been found. [(View Details)](https://app.snyk.io/org/platform-core-video/pr-checks/b46df4d5-2548-44e4-80af-bf2a0dbd0a76?refs=b9b1c0eda0069a8a4d528c17a5b9b00296c3fee1&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/platform-core-video/pr-checks/b46df4d5-2548-44e4-80af-bf2a0dbd0a76/license?refs=b9b1c0eda0069a8a4d528c17a5b9b00296c3fee1&source=prChecksComment)

:white_check_mark: **code/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/platform-core-video/project/21e490d4-e128-4337-a992-0ff196fa8db5/pr-check/e1304328-cea2-40bf-bdda-ba24c7da9227?refs=b9b1c0eda0069a8a4d528c17a5b9b00296c3fee1&source=prChecksComment)


:computer: Catch issues earlier using the plugins for [VS Code](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner), [JetBrains IDEs](https://plugins.jetbrains.com/plugin/10972-snyk-security), [Visual Studio](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner-vs-2022), and [Eclipse](https://marketplace.eclipse.org/content/snyk-security#external-install-button).
